hybrasyl/Objects/Monster.cs

```diff
@@ -35,6 +40,14 @@ public class Monster : Creature, ICloneable
 
         private Spawn _spawn;
 
+        private Creatures.Damage _simpleDamage;
+
+        public int ActionDelay = 800;
+
+        public DateTime LastAction { get; set; }
+        public bool IsHostile { get; set; }
+        public bool ShouldWander { get; set; }
+
```

Comment:
These definitely should be, and they should be using XML values.

Reply:
These are randomly generated at the time of creation. XML will serve no purpose here.

Reply:
IsHostile / ShouldWander are randomly generated? Why? Both of those are behavioral settings that you would want to be able to express in XML.

Reply:
Both are calculated settings derived at spawn time. They should not exist in XML.